base/Makefile

```diff
@@ -249,50 +248,16 @@ ${CLUSTER_GEN}/remote-healthcheck-image: ${CLUSTER_GEN}/healthcheck-image-tagged
 	@  exit 1
 	@fi
 	kubectl config use-context "kctf_${PROJECT}_${ZONE}_${CLUSTER_NAME}" >&2
+	kubectl config set-context --current --namespace="${CHALLENGE_NAME}"
+	kubectl create namespace "${CHALLENGE_NAME}" --dry-run -oyaml | kubectl apply -f -
+	kubectl patch ServiceAccount default --patch "automountServiceAccountToken: false"
 	mkdir -p ${CLUSTER_GEN} >&2
 
 .FORCE:
 
 define DEPLOYMENT_KUSTOMIZATION
 commonLabels:
   app: "$(CHALLENGE_NAME)"
```

Comment:
@sirdarckcat this can be removed as well, right?

Reply:
Which part? Not sure I understood :-)

Reply:
I think I made a mistake highlighting the line. All the code that sets/uses the app=challengename is not needed since app == namespace now, right?

Reply:
ah yea, I think so =D

Reply:
the app thing can't be removed since the deployment needs a selector and that one usually uses the app label